### ISSUE TITLE
abnormal citrulline metabolism

### ISSUE THREAD

**Issue by slaulederkind** [obophenotype/mammalian-phenotype-ontology#2776]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

1. abnormal citrulline metabolism
2. increased plasma citrulline
3. decreased plasma citrulline

>2. Synonyms (e.g., absent spleen)

2. elevated plasma citrulline, high plasma citrulline
3. low plasma citrulline, reduced plasma citrulline

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

1. any anomaly in the chemical reactions and pathways involving citrulline, including metabolic, catabolic and biosynthetic processes
2. any anomaly in the chemical reactions and pathways involving citrulline that yields a net increase in plasma citrulline
3. any anomaly in the chemical reactions and pathways involving citrulline that yields a net decrease in plasma citrulline

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

PMID:16773471 

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

1. abnormal amino acid metabolism (MP:0013242)
2. abnormal citrulline metabolism (new term)
3. abnormal citrulline metabolism (new term)

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

RGD:sjwang
